src/styles/components/buttons.less

```diff
@@ -35,6 +35,29 @@ components/buttons.less
         }
       }
     }
+
+    &-up,
```

Comment:
This doesn't address the problem. It is hard for me to search for the class definition if the class is not defined as a whole string